src/Event.php

```diff
@@ -747,4 +752,24 @@ public function setSpans(array $spans): void
     {
         $this->spans = $spans;
     }
+
+    public function hasAttachments(): bool
```

Comment:
Considering every method in this file has (maybe useless) comments it is nice to also add a docbloc here.